File: server_nodejs/src/models/item_img.js
//Item_img (UID, itemID, link, is_primary)

module.exports = (sequelize, Sequelize) => {
    class ItemImage extends Sequelize.Model {}

    ItemImage.init(
        {
            uid: {
                type: Sequelize.UUID,
                primaryKey: true,
            },
            itemid: {
                type: Sequelize.UUID,
                allowNull: false,
            },
            link: {
                type: Sequelize.STRING,
                allowNull: false,
            },
            is_primary: {
                type: Sequelize.BOOLEAN,
                allowNull: false,
            },
        },
        {
            sequelize,
            modelName: 'item_img',
            timestamps: true,
            paranoid: true,
            underscored: true,
        },
    );

    ItemImage.associate = (models) => {
        ItemImage.belongsTo(models.item, {
            foreignKey: 'itemid',
        });
    };
    return ItemImage;
};

File: server_asp/src/models/ItemImg.cs
using System;
using System.Collections.Generic;

namespace shieldtify.models;

public partial class ItemImg
{
    public Guid Uid { get; set; }

    public Guid Itemid { get; set; }

    public string Link { get; set; } = null!;

    public bool IsPrimary { get; set; }

    public DateTime CreatedAt { get; set; }

    public DateTime UpdatedAt { get; set; }


    public virtual Item Item { get; set; } = null!;
}
